project.go

```diff
@@ -111,7 +111,7 @@ func (p *Project) SetRoot(root string) error {
 		return err
 	}
 
-	p.ResolvedAbsRoot, p.AbsRoot = rroot, root
+	p.ResolvedAbsRoot, p.AbsRoot = rroot, filepath.Clean(root)
```

Comment:
Should we do the same for `rroot`?

Reply:
I think not, rroot from filepath.EvalSymlinks has its internal Clean